github/MainClass.py

```diff
@@ -784,97 +794,3 @@ def get_app(self, slug=github.GithubObject.NotSet):
         else:
             headers, data = self.__requester.requestJsonAndCheck("GET", f"/apps/{slug}")
             return GithubApp.GithubApp(self.__requester, headers, data, completed=True)
-
-
-class GithubIntegration:
```

Comment:
Now for my big question -- is this an API break?

Reply:
Perhaps with a deprecation message as well? Then we don't have to ship it forever.

Reply:
I'm not sure how to add a message when a user imports from the wrong location, do you have something in mind?